getting_started/step_by_step/scripting_first_script.rst

```diff
@@ -343,3 +399,40 @@ Here is the complete ``sprite_2d.gd`` file for reference.
             Position += velocity * (float)delta;
         }
     }
+ 
+ .. code-tab:: cpp C++
+
+    #ifndef MY_SPRITE_2D_H
+    #define MY_SPRITE_2D_H
+
+    #include <godot_cpp/variant/utility_functions.hpp>
+    #include <godot_cpp/classes/sprite2d.hpp>
+    #include <godot_cpp/core/math.hpp>
+
+    namespace CustomNamespace {
+
+    class MySprite2D : public Sprite2D {
+        GDCLASS(MySprite2D, godot::Object)
+
+        int speed = 400;
+        float angular_speed = Math_PI;
+
+    protected:
+        static void _bind_methods() {}
+
+    public:
+        MySprite2D() {
+            godot::UtilityFunctions::print("Hello, world!");
+        }
+
+        void _process(const double p_delta) override {
```

Comment:
```suggestion
        void _process(double p_delta) override {
```
No `const` for simple types

Reply:
Does adding/removing const here make any difference? I just copied the function signature from the source code so I actually don't know.

Reply:
It's per the style, will check if the extension code generates `const` in godot-cpp and if so it should be fixed for simple types

Reply:
It looks like GDExtension does not use const in their _process function, so I will go with that